src/cljs/rems/administration/create_form.cljs

```diff
@@ -74,7 +68,7 @@
 
 (rf/reg-sub ::form (fn [db _]
                      (-> (::form db)
-                         (update :form/fields #(vec (map-indexed (fn [i field] (assoc field :field/id i)) %))))))
+                         (update :form/fields #(vec (map-indexed (fn [i field] (assoc field :field/index i)) %))))))
```

Comment:
should there be a comment explaining the :field/index vs :field/id logic in the frontend?